addons/binding/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/config/coolmasternetConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package org.openhab.binding.coolmasternet.config;
+
+/**
+ * The {@link coolmasternetConfiguration} is responsible for holding
+ * configuration information needed to access/poll the coolmasternet unit
+ *
+ * @author Angus Gratton
+ */
+
+public class coolmasternetConfiguration {
```

Comment:
It seems that you accidentially created the skeleton with "coolmasternet" instead of "CoolMasterNet" as a binding name. This now created all your classes in small caps, which is against Java class naming conventions.
I'd suggest you use Eclipse refactoring support to change them to CoolMasterNet.